### DISCUSSION TITLE
[BUG] No way to set variable_print_min or variable_print_max

### DISCUSSION THREAD

**Discussion by Heratiki** [jschuh/klipper-macros#108] [Q&A]
I've tried several ways to set these variables with no success.  Using variable_print_min: * or variable_print_max: * or with an = between them causes an EOF error. 

`Unable to parse '' as a literal: unexpected EOF while parsing (, line 0)`

**Comment by jschuh** [comment]
I migrated your bug here, since the Q&A discussion is the correct place for help requests.

Without actually seeing the config I don't know what the error is. That's why the bug reporting template and readme both direct you to attach your config. Broadly speaking though, you just need a section like this in your config (substituting your own values and including lines for any other settings you want to change):

```
[gcode_macro _km_options]
variable_print_min: (0,0)
variable_print_max: (220,220)
gcode: # This line is required by Klipper.
```

There's more detail in the [installation instructions](https://github.com/jschuh/klipper-macros#klipper-setup), and you can find all the options in the [globals file](https://github.com/jschuh/klipper-macros/blob/main/globals.cfg#L16).

**Reply by Heratiki** [reply]
I set these variables within my config right under to variable_start_purge_length. I've tried multiple settings of min 2 max 50 or min 5 max 10 etc and it still just purges 30mm of filament into thin air off the front of my bed exactly the same every time. 

Edit: I see where I made my mistake. I'll correct it and give it a shot. Thank you.

**Comment by jschuh** [comment]
Glad you got it sorted.
